.github/workflows/e2e.yaml

```diff
@@ -25,26 +33,30 @@ jobs:
       - name: Build k8s-snap image
         run: |
           cd templates/docker
-          sudo docker build . -t k8s-snap:dev
+          sudo docker build . -t k8s-snap:dev-old --build-arg BRANCH=main --build-arg KUBERNETES_VERSION=v1.29.6
+          sudo docker build . -t k8s-snap:dev-new --build-arg BRANCH=main
```

Comment:
should this be in a wrapper script? where you build all necessary images? this version will need to be updated eventually, it would rather not be a hardcoded value on a github action